src/main/java/ch/njol/skript/util/ReplacementTypes.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.util;
+
+public enum ReplacementTypes {
+	ALL,
+	FIRST,
+	LAST
```

Comment:
just to print it better for the toString methods in the syntax elements
```suggestion
	
	ALL,
	FIRST,
	LAST
	
	@Override
	public String toString() {
		return super.toString().toLowerCase();
	}
	
```